.github/workflows/integration.yaml

```diff
@@ -54,6 +54,7 @@ jobs:
         uses: charmed-kubernetes/actions-operator@main
         with:
             provider: microk8s
+            # TODO: Change to 1.22 once https://github.com/juju/python-libjuju/issues/684 is fixed
             channel: 1.21/stable
```

Comment:
There is desire to use dex outside of just kubeflow.. It would be nice if we validated it against all versions of k8s.

Reply:
Should be easy enough to add a `matrix` block to this to cover recent versions of MicroK8s :)